build.sbt

```diff
@@ -1,6 +1,13 @@
+import sbtrelease._
+import ReleaseStateTransformations._
+
+releaseSettings
+
+sonatypeSettings
+
 name in ThisBuild := "feature-switching"
 
-version in ThisBuild := "0.13-SNAPSHOT"
+version in ThisBuild := "1.0-SNAPSHOT"
```

Comment:
move version out of this file and into a `version.sbt` file in the root.

This is where the release plugin will be looking for it.